src/lib/containers/conversation.tsx

```diff
@@ -55,6 +58,16 @@ interface Props extends ViewProperties {
 export default class Conversation extends React.Component<Props, any> {
   render() {
     const partnerName = "Patrick Parrish Gallery"
+    // tslint:disable-next-line:max-line-length
+    const messageBody =
+      "Hi, I'm interested in purchasing this work. Could you please provide more information about the piece, including price?"
```

Comment:
I'm going to leave this hard-coded for now because I'm working on the metaphysics side of this at the moment and it'll be a lot nicer to deal with once we can use Relay here 😄